docs/kubernetes.md

```diff
@@ -56,3 +56,109 @@ Pour accéder à votre cluster :
 Plus de détails sur l'administration kube avec k9s sur [cet article](https://opensource.com/article/20/5/kubernetes-administration).
 
 Une sonde [sentry-kubernetes](https://github.com/getsentry/sentry-kubernetes) est installée sur le cluster et permet de remonter toutes les erreurs : CronJob failed, probes... C'est une source d'information préciseuse quand quelque chose ne fonctionne pas dans vos déploiements. L'accès doit être demandé à la team SRE.
+
+## Variable d'environnement dans Kubernetes
+
+On vous recommande de récupérer vos variables d'environnement avec `envFrom`.
+
+```yaml
+# [...]
+  envFrom:
+    - configMapRef:
+        name: app-env
+    - secretRef:
+        name: app-env
+```
+
+### Variables de configuration
```

Comment:
```suggestion
### ConfigMap : Variables de configuration
```

Reply:
Done in https://github.com/SocialGouv/support/commit/114e8164140e4418ae79896c7a4b3ef57c6a933d